batch/batch/batch.py

```diff
@@ -256,261 +98,21 @@ def job_record_to_dict(record, name):
-        env.append({'name': 'KUBECONFIG',
-                    'value': '/.kube/config'})
-
-    if format_version.has_full_spec_in_gcs():
-        token, start_job_id = await SpecWriter.get_token_start_id(db, batch_id, job_id)
-    else:
-        token = None
-        start_job_id = None
-
-    return {
-        'batch_id': batch_id,
-        'job_id': job_id,
-        'format_version': format_version.format_version,
-        'token': token,
-        'start_job_id': start_job_id,
-        'user': record['user'],
-        'gsa_key': gsa_key,
-        'job_spec': job_spec
-    }
-
-
-async def schedule_job(app, record, instance):
-    assert instance.state == 'active'
-
-    log_store = app['log_store']
-    db = app['db']
-
-    batch_id = record['batch_id']
-    job_id = record['job_id']
-    attempt_id = record['attempt_id']
-    format_version = BatchFormatVersion(record['format_version'])
-
-    id = (batch_id, job_id)
-
-    try:
-        try:
-            body = await job_config(app, record, attempt_id)
-        except Exception:
-            log.exception('while making job config')
-            status = {
-                'version': STATUS_FORMAT_VERSION,
-                'worker': None,
-                'batch_id': batch_id,
-                'job_id': job_id,
-                'attempt_id': attempt_id,
-                'user': record['user'],
-                'state': 'error',
-                'error': traceback.format_exc(),
-                'container_statuses': {k: None for k in tasks}
-            }
-
-            if format_version.has_full_status_in_gcs():
-                await log_store.write_status_file(batch_id, job_id, attempt_id, json.dumps(status))
-
-            db_status = format_version.db_status(status)
-            resources = []
-
-            await mark_job_complete(app, batch_id, job_id, attempt_id, instance.name,
-                                    'Error', db_status, None, None, 'error', resources)
-            raise
-
-        log.info(f'schedule job {id} on {instance}: made job config')
-
-        try:
-            async with aiohttp.ClientSession(
-                    raise_for_status=True, timeout=aiohttp.ClientTimeout(total=2)) as session:
-                url = (f'http://{instance.ip_address}:5000'
-                       f'/api/v1alpha/batches/jobs/create')
-                await session.post(url, json=body)
-                await instance.mark_healthy()
-        except aiohttp.ClientResponseError as e:
-            await instance.mark_healthy()
-            if e.status == 403:
-                log.info(f'attempt already exists for job {id} on {instance}, aborting')
-            raise e
-        except Exception:
-            await instance.incr_failed_request_count()
-            raise
-
-        log.info(f'schedule job {id} on {instance}: called create job')
-
-        rv = await db.execute_and_fetchone(
-            '''
-CALL schedule_job(%s, %s, %s, %s);
+async def cancel_batch_in_db(db, batch_id, user):
```

Comment:
This is just a reorganization of the functions in this file. There were no changes from the code on the left side of the diff.